packages/js/product-editor/src/components/variations-table/variations-table.tsx

```diff
@@ -417,63 +414,14 @@ export function VariationsTable() {
 						<div className="woocommerce-product-variations__actions">
 							{ variation.status === 'private' && (
 								<Tooltip
+									// @ts-expect-error className is missing in TS, should remove this when it is included.
+									className="woocommerce-attribute-list-item__actions-tooltip"
 									position="top center"
 									text={ NOT_VISIBLE_TEXT }
 								>
-									<Button
-										className="components-button--hidden"
-										aria-label={
-											isUpdating[ variation.id ]
-												? UPDATING_TEXT
-												: NOT_VISIBLE_TEXT
-										}
-										aria-disabled={
-											isUpdating[ variation.id ]
-										}
-										onClick={ () =>
-											handleVariationChange(
-												variation.id,
-												{ status: 'publish' }
-											)
-										}
-									>
-										{ isUpdating[ variation.id ] ? (
-											<Spinner />
-										) : (
-											<HiddenIcon />
-										) }
-									</Button>
-								</Tooltip>
-							) }
-
-							{ variation.status === 'publish' && (
-								<Tooltip
-									position="top center"
-									text={ VISIBLE_TEXT }
-								>
-									<Button
-										className="components-button--visible"
-										aria-label={
-											isUpdating[ variation.id ]
-												? UPDATING_TEXT
-												: VISIBLE_TEXT
-										}
-										aria-disabled={
-											isUpdating[ variation.id ]
-										}
-										onClick={ () =>
-											handleVariationChange(
-												variation.id,
-												{ status: 'private' }
-											)
-										}
-									>
-										{ isUpdating[ variation.id ] ? (
-											<Spinner />
-										) : (
-											<VisibleIcon />
-										) }
-									</Button>
+									<div>
```

Comment:
It's weird having to add an additional div for this to work... But I tested and it really seems necessary. Any idea why?

Reply:
Yeah I was confused by that as well. My guess is that the Tooltip component doesn't actually wrap the content it-self (maybe?? ), so it can't calculate the size correctly.